### ISSUE TITLE
Remove `Document` wrapper from JSON and YAML examples

### ISSUE THREAD

**Issue by tsteenbe** [spdx/spdx-spec#400]
Both of the SPDX JSON and YAML examples have a `Document` wrapper for the SpdxDocument while this may make sense for XML and RDF this causes an issue in JSON and YAML as there is now no easy way to have multiple SPDX documents in the same file.

Propose to remove `Document` wrapper for the JSON and YAML formats.

[1] https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXYAMLExample-2.2.spdx.yaml#L2
[2] https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXJSONExample-v2.2.spdx.json#L2

**Comment by goneall** [comment]
Let's discuss - I'm not sure I agree that it is so difficult to have multiple SPDX documents in one file - you would just wrap each document.  Wrapping seems to be the approach used in some of the JSON examples: https://json.org/example.html

**Comment by tsteenbe** [comment]
Aware of the JSON.org example that use wrapper help people who know XML to learn JSON.
In the unwritten rule in the JSON / YAML community is that you only use a wrapper element if there other elements at root level or if you are planning to add other elements in the future. 

In this case SPDX document will be the only element at the root so wrapper element is just extra baggage that should be removed.

Having `Document` will also make it more difficult for JSON/YAML creators and consumer to handle SPDX as items are represented the same way in code. As a consumer it now harder to loop over a response with multiple docs - I can't you can just ‘slurp in the JSON’ and you immediately have your domain object but I first have to remove `Document`. Same for creators they will have to artificially insert `Document` when they serialize their classes

**Comment by tsteenbe** [comment]
>  It makes it easier for me to group the fields and makes it more human readable when combined with other JSON elements. 

JSON and YAML have implicit object type definition that why https://json-schema.org/ exists. If a document creator wants to include SPDX documents in a larger document then it that creator's responsibility to wrap the SPDX document(s) in field within their output.

We already made the decision in the SPDX tech call of November 26, 2019 to remove the Document, Package and File wrappers see https://github.com/spdx/spdx-spec/issues/157 / https://wiki.spdx.org/view/Technical_Team/Minutes/2019-11-19.

@zvr  As requested some examples below with and without the `Document` wrapper.

Current spec with `Document` wrapper:

```
{
  "Document" : {
    "SPDXID" : "SPDXRef-DOCUMENT",
    "spdxVersion" : "SPDX-2.2",
    "creationInfo" : {
      "comment" : "",
      "created" : "2010-01-29T18:30:22Z",
      "creators" : [ "Tool: SomeTool-1.0", "Organization: ACME", "Person: Jane Doe ()" ],
      "licenseListVersion" : "3.8"
    },
    "name" : "SPDX-Tools-v2.0",
    "dataLicense" : "CC0-1.0",
    "comment" : "",
    "externalDocumentRefs" : [ {
      "externalDocumentId" : "DocumentRef-spdx-tool-1.2",
      "checksum" : {
        "algorithm" : "SHA1",
        "checksumValue" : "d6a770ba38583ed4bb4525bd96e50461655d2759"
      },
      "spdxDocument" : "http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82C3301"
    } ],
    "hasExtractedLicensingInfos" : [],
    "annotations" : [],
    "documentNamespace" : "http://spdx.org/spdxdocs/spdx-example-444504E0-4F89-41D3-9A0C-0305E82C3301",
    "documentDescribes" : [],
    "packages" : [],
    "files": [],
    "snippets": [],
  }
}
```

without `Document` wrapper:

```
{
  "SPDXID" : "SPDXRef-DOCUMENT",
  "spdxVersion" : "SPDX-2.2",
  "creationInfo" : {
    "comment" : "",
    "created" : "2010-01-29T18:30:22Z",
    "creators" : [ "Tool: SomeTool-1.0", "Organization: ACME", "Person: Jane Doe ()" ],
    "licenseListVersion" : "3.8"
  },
  "name" : "SPDX-Tools-v2.0",
  "dataLicense" : "CC0-1.0",
  "comment" : "",
  "externalDocumentRefs" : [ {
    "externalDocumentId" : "DocumentRef-spdx-tool-1.2",
    "checksum" : {
      "algorithm" : "SHA1",
      "checksumValue" : "d6a770ba38583ed4bb4525bd96e50461655d2759"
    },
    "spdxDocument" : "http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82C3301"
  } ],
  "hasExtractedLicensingInfos" : [],
  "annotations" : [],
  "documentNamespace" : "http://spdx.org/spdxdocs/spdx-example-444504E0-4F89-41D3-9A0C-0305E82C3301",
  "documentDescribes" : [],
  "packages" : [],
  "files": [],
  "snippets": [],
}
```

If you have a file with multiple SPDX documents you the below code:

```
[
  {
    "Document" : {
       "SPDXID" : "SPDXRef-DOCUMENT-A",
        ...
    }
  },
  {
    "Document" : {
       "SPDXID" : "SPDXRef-DOCUMENT-B",
        ...
    }
  }
]
```

Without a `Document` wrapper you get below code, notice how you can directly loop over the array and access each document's properties.

```
[
  {
     "SPDXID" : "SPDXRef-DOCUMENT-A",
      ...
  },
  {
     "SPDXID" : "SPDXRef-DOCUMENT-B",
      ...
  }
]

**Comment by goneall** [comment]
Based on issue #157 stating that the wrapper for Document should be removed, I am now thinking this is a bug and the example should be updated.  Having this issue resolution to refer to resolves my concern that we would revisit the decision since it looks like it was already discussed.

It looks like I removed the wrappers for the file and package but did not remove the Document wrapper.

**Comment by zvr** [comment]
If it matters, I also support the removal of the top-level `Document` wrapper.

**Comment by goneall** [comment]
Fixed in #416
